lib/std/Target.zig

```diff
@@ -979,26 +979,38 @@ pub const Cpu = struct {
     };
 
     pub const Arch = enum {
+        alpha,
```

Comment:
Alpha technically allows big endian implementations. To my knowledge, no such implementation has ever existed, and no software that supports Alpha supports it in a big endian configuration. So no `alphaeb` here.